### PR TITLE
Use the copy to clipboard component

### PR DESCRIPTION
**Pull request by tijmenb**
See commits for details.

https://trello.com/c/hAfZNnxt

### CODE REVIEW

File: app/views/components/_page_preview.html.erb [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <%= render "govuk_publishing_components/components/details", title: "Share" do %>
-  <p>Copy and send this link to someone and they’ll be able to preview your draft on GOV.UK.</p>
+  <%= render "govuk_publishing_components/components/copy_to_clipboard",
+    label: "Copy and send this link to someone and they’ll be able to preview your draft on GOV.UK.",
+    copyable_content: url,
+    button_text: "Copy link" %>
 
   <%= link_to url, url %>
```
Comment (alex-ju) [inline app/views/components/_page_preview.html.erb]: I think we don't need this line anymore.

File: config/locales/en/global.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+en:
```
Comment (benthorner) [inline config/locales/en/global.yml]: We decided not to use translations for any components, as they may be moved out of this repo in future into the main publishing components one, where the translations wouldn’t work.

Comment (tijmenb) [reply config/locales/en/global.yml]: This string isn't used inside the component, just passed into it.

Comment (benthorner) [reply config/locales/en/global.yml]: Since this is an interactive element, you don't need to use a translation for it.

**Pull request merged**